repooler.py

```diff
@@ -268,11 +288,12 @@ def correct_numbers(lane_maps, clusters_expr, ideal_ratios, req_lanes, total_lan
                     
     return acc_ratios
 
-def generate_output(project, destid, total_lanes, req_lanes, lane_maps, acc_ratios):
-    #Gathers the container id and well name for all samples in project
+""""Gathers the container id and well name for all samples in project"""
```

Comment:
By convention, docstrings go below the `def some_function():` declaration. No biggie.